model-archiver/README.md

```diff
@@ -139,6 +139,7 @@ A serialized file (.pt or .pth) should be a checkpoint in case of torchscript an
 ### Handler
 
 Handler can be TorchServe's inbuilt handler name or path to a py file to handle custom TorchServe inference logic. TorchServe supports the following handlers out of box:
+0. `base_handler`
```

Comment:
I don't believe this renders correctly a list, just start from 1

Reply:
It seems to render correctly on GitHub: https://github.com/pytorch/serve/blob/de0868f53b36782f78da6491c3e77603b0a57ff6/docs/custom_service.md#extend-default-handlers

But I can change it to `1.` if needed.